localtypings/pxtarget.d.ts

```diff
@@ -365,7 +365,8 @@ declare namespace pxt {
         checkForHwVariantWebUSB?: boolean; // check for hardware variant using webusb before compiling
         shareFinishedTutorials?: boolean; // always pop a share dialog once the tutorial is finished
         leanShare?: boolean; // use leanscript.html instead of script.html for sharing pages
-        nameProjectFirst?: boolean;
+        nameProjectFirst?: boolean; // prompt user to name project when creating new one
+        newProjectOptions?: boolean; // include 'options' menu when creating a new project
```

Comment:
more specific flag maybe: chooseLanguageRestrictionOnNewProject . it's verbose but self-explaijning.